tests/routes/test_search.py

```diff
@@ -245,6 +245,33 @@ def _create_family_structures(
     documents[doc_id] = family_document
 
 
+@pytest.mark.search
+def test_slug_is_from_family_document(test_opensearch, client, test_db, monkeypatch):
+    monkeypatch.setattr(search, "_OPENSEARCH_CONNECTION", test_opensearch)
+    _populate_search_db_families(test_db)
+    search_endpoint = f"{SEARCH_ENDPOINT}?group_documents=True"
+
+    page1_response = client.post(
+        search_endpoint,
+        json={
+            "query_string": "and",
+            "exact_match": False,
+            "limit": 2,
+            "offset": 0,
+        },
+    )
+    assert page1_response.status_code == 200
+
+    page1_response_body = page1_response.json()
+    fam1 = page1_response_body["families"][0]
+    doc1 = fam1["family_documents"][0]
+    slug = doc1["document_slug"]
+    print()
+    print(">>>> " + slug)
+    print()
+    assert doc1["document_slug"].startswith("fd_") and "should be from FamilyDocument"
```

Comment:
Is this actually a rule that document slugs must start with fd_ ? 

I can't see an assertion of this in the ingest script?

Reply:
Nope this is a test "feature" see line 236 above ... just so I can ensure they are correct